tests/parameters/fixtures/active-choices-param001.yaml

```diff
@@ -0,0 +1,29 @@
+parameters:
+  - active-choices:
+      name: lorem
+      description: ipsum
+      script:
+          groovy: |-
+              return [
+                  'param1',
+                  'param2'
+              ]
+          use-groovy-sandbox: 'true'
```

Comment:
According to the parameter documentation it's bool, not str.

Reply:
I was having errors trying to keep it as a bool, that was why I have put it as string as it would not affect the XML, but I agree that it was a bad practice. I made it as bool now thanks to your comment!